### PR TITLE
Create dropins to override service debug level on both manager and storage servers

### PR DESCRIPTION
**Pull request by johnsonw**
- Account for docker services
- Account for systemd services

Signed-off-by: johnsonw <wjohnson@whamcloud.com>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1872)

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -284,7 +284,7 @@
 
 GUNICORN_PID_PATH = "/var/run/gunicorn.pid"
 
-LOG_LEVEL = logging.INFO
+LOG_LEVEL = os.getenv("LOG_LEVEL", logging.INFO)
```
Comment (jgrund) [inline settings.py:287]: Is it a problem if LOG_LEVEL is a string?

Comment (johnsonw) [reply settings.py]: i set this to always be an int now.

**Comment by johnsonw** [comment]
sure that makes sense. Can we do something similar on the storage servers or will I need to do it the way I have it now?

**Comment by johnsonw** [comment]
This command will print out all of the services for the manager along with their environment variables. We should see RUST_LOG=debug and LOG_LEVEL=10 for most services (a couple of exceptions).

```
ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null adm "systemctl list-dependencies iml-manager.target" | tail -n +2 | awk '{print$2}' | awk '{print substr($1, 3)}' | grep -v iml-settings-populator.service | xargs -I {} systemctl show --property MainPID {} | cut -d'=' -f2 | xargs -I {} bash -c "echo 'Pid'; echo {}; echo 'Service:';ps -p {} -o unit=;echo 'Environment Variables:'; cat /proc/{}/environ; echo ; echo;"
Pid
861
Service:
grafana-server.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binNOTIFY_SOCKET=/run/systemd/notifyHOME=/usr/share/grafanaLOGNAME=grafanaUSER=grafanaSHELL=/sbin/nologinGRAFANA_USER=grafanaGRAFANA_GROUP=grafanaGRAFANA_HOME=/usr/share/grafanaLOG_DIR=/var/log/grafanaDATA_DIR=/var/lib/grafanaMAX_OPEN_FILES=10000CONF_DIR=/etc/grafanaCONF_FILE=/etc/grafana/grafana.iniRESTART_ON_UPGRADE=truePLUGINS_DIR=/var/lib/grafana/pluginsPROVISIONING_CFG_DIR=/etc/grafana/provisioningPID_FILE_DIR=/var/run/grafana

Pid
8595
Service:
iml-action-runner.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binLISTEN_PID=8595LISTEN_FDS=1RUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8591
Service:
iml-agent-comms.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8562
Service:
iml-api.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8596
Service:
iml-corosync.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8561
Service:
iml-device.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8371
Service:
iml-gunicorn.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binNOTIFY_SOCKET=/run/systemd/notifyUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8549
Service:
iml-http-agent.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8546
Service:
iml-job-scheduler.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8574
Service:
iml-lustre-audit.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8547
Service:
iml-mailbox.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8548
Service:
iml-ntp.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8605
Service:
iml-ostpool.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8571
Service:
iml-plugin-runner.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8573
Service:
iml-postoffice.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8550
Service:
iml-power-control.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8560
Service:
iml-rust-stats.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=debugDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=TrueAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=whamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672LOG_LEVEL=10

Pid
8552
Service:
iml-syslog.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binUSE_STRATAGEM=TrueBRANDING=whamcloudRUST_LOG=debugLOG_LEVEL=10

Pid
8572
Service:
iml-warp-drive.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binRUST_LOG=infoDB_NAME=chromaIML_API_PROXY_PASS=http://127.0.0.1:8004INFLUXDB_IML_STATS_LONG_DURATION=52wAPI_USER=apiPROXY_HOST=127.0.0.1ALLOW_ANONYMOUS_READ=trueDB_PASSWORD=SITE_ROOT=/usr/share/chroma-managerMAILBOX_PORT=8891AMQP_BROKER_VHOST=chromavhostINFLUXDB_STRATAGEM_SCAN_DB=iml_stratagem_scansDB_USER=chromaAMQP_BROKER_PASSWORD=chroma123VERSION=6.0.0-1BUILD=HTTP_AGENT2_PORT=8003AMQP_BROKER_HOST=localhostUSE_STRATAGEM=falseAMQP_BROKER_URL=amqp://chroma:chroma123@localhost:5672/chromavhostHTTP_AGENT2_PROXY_PASS=http://127.0.0.1:8003MAILBOX_PATH=/var/spool/iml/mailboxIML_API_PORT=8004IS_RELEASE=trueSERVER_HTTP_URL=https://adm.local:443AMQP_BROKER_USER=chromaINFLUXDB_PORT=8086INFLUXDB_IML_DB=imlLOG_PATH=/var/log/chromaDEVICE_AGGREGATOR_PORT=8008SOURCE_MAP_PATH=NoneWARP_DRIVE_PORT=8890INFLUXDB_IML_STATS_DB=iml_statsDB_HOST=BRANDING=WhamcloudSFA_ENDPOINTS=API_KEY=6bbf8fe491bd36e4e36ca221dcc7b426f5acf7bdAMQP_BROKER_PORT=5672

Pid
864
Service:
influxdb.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binHOME=/var/lib/influxdbLOGNAME=influxdbUSER=influxdbSHELL=/bin/falseINFLUXDB_DATA_QUERY_LOG_ENABLED=falseINFLUXDB_REPORTING_DISABLED=trueINFLUXDB_HTTP_LOG_ENABLED=false

Pid
1429
Service:
nginx.service
Environment Variables:
ocess /usr/sbin/nginx

Pid
878
Service:
postgresql-9.6.service
Environment Variables:
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binNOTIFY_SOCKET=/run/systemd/notifyHOME=/var/lib/pgsqlLOGNAME=postgresUSER=postgresSHELL=/bin/bashPGDATA=/var/lib/pgsql/9.6/data/PG_OOM_ADJUST_FILE=/proc/self/oom_score_adjPG_OOM_ADJUST_VALUE=0

Pid
8442
Service:
rabbitmq-server.service
Environment Variables:
EMU=beamSHELL=/sbin/nologinBINDIR=/usr/lib64/erlang/erts-5.10.4/binUSER=rabbitmqROOTDIR=/usr/lib64/erlangPATH=/usr/lib64/erlang/erts-5.10.4/bin:/usr/lib64/erlang/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binPWD=/var/lib/rabbitmqLANG=en_US.UTF-8PROGNAME=erlNOTIFY_SOCKET=/run/systemd/notifyHOME=/var/lib/rabbitmqSHLVL=0LOGNAME=rabbitmq
```

**Comment by johnsonw** [comment]
Here are the environment variables for the rust-iml-agent service on one of the storage servers:

```
[root@oss1 ~]# systemctl show --property MainPID rust-iml-agent
MainPID=820
[root@oss1 ~]# cat /proc/820/environ
LANG=en_US.UTF-8PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/binIML_MANAGER_URL=https://adm.local:443RUST_LOG=debugPRIVATE_PEM_PATH=/etc/iml/private.pemCRT_PATH=/etc/iml/self.crtPFX_PATH=/etc/iml/identity.pfxAUTHORITY_CRT_PATH=/etc/iml/authority.crtPOSTMAN_CONF_PATH=/etc/iml/postman.confSOCK_DIR=/run/imlLTUER_CONF_PATH=/etc/iml/ltuer.confLAMIGO_CONF_PATH=/etc/lamigo/{fs}-{mdt}.confLPURGE_CONF_PATH=/etc/lpurge/{fs}/{ost}.confLOG_LEVEL=10
```

**Pull request closed without merging**